src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {

```

Comment:
According to the checklist, don't use abstract classes to set behavior for classes. Consider using an interface to define the behavior and then implement it in the specific classes.